CryptoPrices/Sources/Data/Sources/NetworkExtension/CoinRoute.swift

```diff
@@ -11,19 +11,35 @@ import PilotType
 public enum CoinRoute {
 
     // TODO: Update when implement
-    case myCoins
+    case myCoins(MyCoinsParameters)
     case trendingCoins
     case coinDetail
 }
 
 extension CoinRoute: Route {
 
-    // TODO: Update when implement
     // swiftlint:disable:next force_unwrapping
     public var baseURL: URL { .init(string: "https://api.coingecko.com/api/v3")! }
-    public var path: String { "" }
+
+    public var path: String {
+        switch self {
+        case .myCoins: return "/coins/market"
+            // TODO: Update when implement
+        default: return ""
+        }
+    }
+
     public var httpMethod: HttpMethod { .get }
     public var httpHeaders: HttpHeaders { .empty }
-    public var parameters: Parameters? { nil }
+
+    public var parameters: Parameters? {
+        switch self {
+        case let .myCoins(parameters): return parameters.encoded()
+        default:
+            // TODO: Update when implement
+            return nil
+        }
+    }
```

Comment:
Small code refactor to be consistent with `path` 🙏 
```suggestion
    public var parameters: Parameters? {
        switch self {
        case let .myCoins(parameters): return parameters.encoded()
           // TODO: Update when implement
        default: return nil
        }
    }
```

Reply:
Nice catch with attention to detail 😄 Will update soon!

Reply:
Updated with [#062134e](https://github.com/nimblehq/nimble-crypto-ios/pull/33/commits/062134eefb9111754b288ed6614af17451aa4eed)!